prime-dotnet-webapi/Services/EnrolleePaperSubmissionService.cs

```diff
@@ -190,21 +191,21 @@ public async Task UpdateAgreementAsync(int enrolleeId, PaperEnrolleeAgreementVie
             await _enrolleeAgreementService.AcceptCurrentEnrolleeAgreementAsync(enrolleeId);
         }
 
-        public async Task AddEnrolleeAdjudicationDocumentsAsync(int enrolleeId, int adminId, IEnumerable<Guid> documentGuids)
+        public async Task AddEnrolleeAdjudicationDocumentsAsync(int enrolleeId, int adminId, IEnumerable<PaperEnrolleeDocumentViewModel> documents)
         {
-            foreach (var guid in documentGuids)
+            foreach (var document in documents)
             {
-                var filename = await _documentClient.FinalizeUploadAsync(guid, DestinationFolders.EnrolleeAdjudicationDocuments);
+                var filename = await _documentClient.FinalizeUploadAsync(document.DocumentGuid, DestinationFolders.EnrolleeAdjudicationDocuments);
 
                 if (string.IsNullOrWhiteSpace(filename))
                 {
-                    _logger.LogError($"Could not finalize document {guid}");
+                    _logger.LogError($"Could not finalize document {document}");
                     continue;
                 }
 
                 var adjudicationDocument = new EnrolleeAdjudicationDocument
                 {
-                    DocumentGuid = guid,
+                    DocumentGuid = document.DocumentGuid,
                     EnrolleeId = enrolleeId,
                     Filename = filename,
                     UploadedDate = DateTimeOffset.Now,
```

Comment:
Document type is not being set; presumably accidentally remove in cleanup.

Reply:
Fixed